pkg/testing/gadgetrunner/gadgetrunner.go

```diff
@@ -215,7 +215,40 @@ func (g *GadgetRunner[T]) RunGadget() {
 	require.NoError(g.testCtx, err, "running gadget error")
 }
 
+const (
+	defaultDomain      = "ghcr.io"
+	officialRepoPrefix = "inspektor-gadget/gadget/"
```

Comment:
IMO these values should be taken from https://github.com/inspektor-gadget/inspektor-gadget/blob/004c8916cb6e1928d567bff1ae3551ecbf3a9559/pkg/oci/oci.go#L92-L93